### PR TITLE
Fix dev setup

### PR DESCRIPTION
**Pull request by janfjohannes**

### CODE REVIEW

**Comment by janfjohannes** [comment]
@samuelstroschein  i think this is a really good compromise to keep most of your ideal external contributor flow, but also be not too error prone for inlang team members :)

**Comment by janfjohannes** [comment]
it seems CI is also relying on the public dev environment on doppler in some parts instead of using the CI environment everywhere, which is a good catch of the new setup.

File: inlang/source-code/env-variables/build.ts [hunk 1]
```diff
@@ -17,16 +16,11 @@ config({ path: rootEnvFilePath, override: false })
 const { error: errors } = validateEnvVariables({ forProduction: false })
 
 if (errors) {
-	console.error("💡 Some env variables are not defined. Fetching public env variables remotely...")
+	console.error(
+		"❗️❗️ Some env variables are not defined or do not pass validation. If you are an external developer, please run `npm run fetchPublicEnv` to get the latest version, if you are part of the inlang team, please check your doppler setup."
```
Comment (samuelstroschein) [inline inlang/source-code/env-variables/build.ts:20]: `npm run fetchPublicEnv` will break in the monorep setup, assuming that developers start from the root of the repo. 

`npm run fetchPublicEnv -w @inlang/env-variables` will likely

**Review by samuelstroschein — APPROVED** [review]

File: inlang/source-code/env-variables/build.ts (hunk 1 above)
Comment (janfjohannes) [reply inlang/source-code/env-variables/build.ts:20]: @samuelstroschein  i added the command to the root of the repo too:
"fetchPublicEnv": "npx nx run @inlang/env-variables:fetchPublicEnv",

**Pull request merged**